### ISSUE TITLE
Allow for garbage collection to be adjusted by the player

### ISSUE THREAD

**Issue by fritogotlayed** [DUWS-R-Team/DUWS-R#37]
We've had report of the game getting sluggish for players after an hour or so of gameplay. In an effort to help players fine-tune the mission to stay performing well on their system it may be worth while to expose the options for the cleanup routine to players. We will most likely need some design around this but currently I am thinking a new menu screen under the "settings" area to help with this. 

The understood trade-off here is that looting from fallen opfor may suffer since the routine may clean items up based on the players settings

EXPECTED OUTCOME
A new dialog is displayed after the user clicks the "settings" option from the radial menu. This would allow you to either drop into the **new** dialog that allows tweaking of the settings to garbage collection *or* the **existing** menu that lets the user configure the grass draw distance and excreta.

**Comment by bolbies** [comment]
The reason why the game slows down so much later on is because once you capture zones, it'll spawn soldiers, helicopters, jets, etc based on your army power just like how the HQ sends out task forces and the enemy sends attack waves. More units=worse performance. Then, to add to that, most of those units being spawned from your captured zones are on patrols which hits even harder. The easiest ways to last longer without performance drop is to reduce BLUFOR army power, and instead of spawning and fortifying FOB's, either send a fireteam to sit in the zone to "keep" it, or do what I do and just get a high command heli to guard it. We could easily decrease the time between cleanup intervals but that's pretty much it. It's mostly arma itself really :/.

**Comment by fritogotlayed** [comment]
I figured a good chunk of it was ArmA honestly. Thanks a ton for the analysis!

**Comment by rlex** [comment]
AI limiter in duws modified was used for that purposes. Idea was to spawn units on some range around player, not all units in same time, AFAIK

**Comment by bolbies** [comment]
Yeah that and maybe we can also implement headless clients at some point. Maybe once Zorrobyte fixes up his cache system we can put that in too but as of right now it really drops performance late game.
